docs/developer/release/1-create-release-branch.md

```diff
@@ -0,0 +1,38 @@
+# Create Release Branch
+
+A single release branch is created for every major or minor release. That release
+branch is then used for all Release Candidates, the Stable Release, and all
+Patch Releases.
+
+## Release Version Prefix
+
+The `[release version prefix]` can be determined by looking at the last release
+version and adding to it. Also, drop off the 3rd number as shown in the examples below.
+
+- v0.30.0 -> v0.31
+- v0.30.3 -> v0.31
+
+For a major version we jump up the first number as shown in the examples below.
+
+- v0.10.0 -> v1.0
+- v0.31.0 -> v1.0
+
+*NOTE: This value will be referred to as `[release version prefix]` in this documentation*
+
+## Steps
+
+1. Gather consensus on which commit should be used as a base for the release
```

Comment:
I would change this to `Determine which commit` IMO the release shepherd has control and doesnt necessarily need consensus.